pkg/helper/taskrun/description/taskrun_description.go

```diff
@@ -23,6 +23,11 @@ const templ = `{{decorate "bold" "Name"}}:	{{ .TaskRun.Name }}
 {{decorate "bold" "Service Account"}}:	{{ .TaskRun.Spec.ServiceAccountName }}
 {{- end }}
 
+{{- $timeout := getTimeout .TaskRun -}}
+{{- if ne $timeout "" }}
+{{decorate "bold" "Timeout\n"}} {{ .TaskRun.Spec.Timeout.Duration.String }}
```

Comment:
We also want it to follow the pattern of what is in the `.golden` files for output:

```
Name:        tr-1
Namespace:   ns
Task Ref:    t1
Timeout:     1h0m0s
```

So

```
{{decorate "bold" "Timeout"}}: {{ .TaskRun.Spec.Timeout.Duration.String }}
```

Reply:
Done

Reply:
Done